sockeye/train.py

```diff
@@ -555,9 +555,17 @@ def define_optimizer(args, lr_scheduler_instance) -> Tuple[str, Dict, str]:
                         "learning_rate": args.initial_learning_rate}
     if lr_scheduler_instance is not None:
         optimizer_params["lr_scheduler"] = lr_scheduler_instance
-    clip_gradient = none_if_negative(args.clip_gradient)
-    if clip_gradient is not None:
-        optimizer_params["clip_gradient"] = clip_gradient
+    gradient_clipping_threshold = none_if_negative(args.clip_gradient)
+    if gradient_clipping_threshold is None:
+        logger.info("Gradient clipping threshold set to negative value. Will not perform gradient clipping.")
+        gradient_clipping_type = "none"
```

Comment:
why not use the constant defined in constants.py?

Reply:
good point. I missed this line.